kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -51,14 +51,14 @@
       <tbody>
         <tr v-for="user in visibleUsers">
           <!-- Student Name field -->
-          <th scope="row" class="table-cell">
+          <td scope="row" class="table-cell">
             {{user.full_name}}
 
             <!-- Logic for role tags -->
             <span class="user-role" v-for="role in user.roles">
               {{role.kind | capitalize}}
             </span>
-          </th>
+          </td>
```

Comment:
this was a change that @radinamatic had requested - was it not working as expected?